samples/powerpoint/custom-functions/search-wikipedia-api.yaml

```diff
@@ -0,0 +1,221 @@
+order: 1
+id: powerpoint-custom-functions-search-wikipedia-api
+name: Search Wikipedia
+description: Search Wikipedia based on the selected text in the presentation.
+author: cakriwut
+host: POWERPOINT
+api_set: {}
+script:
+    content: |
+        declare var moment: any;
+        $("#search").click(run);
+
+        async function run() {
+          Office.context.document.getSelectedDataAsync(Office.CoercionType.Text, {}, getSelectedText);
+        }
+
+        function getSelectedText(result) {
+          $("#result").empty();
+          $("#result").append('<ul class="ms-List"></ul>');
+          if (result.status === "succeeded") {
+            searchWiki(result.value);
+          } else {
+            console.log("failed");
```

Comment:
Please expand the error statement to explain what part of the call failed, as best as you're able.

Reply:
Updated.